Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.7-alpine
+WORKDIR /app
+COPY . ./
+RUN apk update
```

Comment:
If you move the apk and gem calls before the workdir and copy lines, it'll improve the build times if you modify a file in your local folder (layer reuse).

Reply:
Oh, that's good to know!  I'll make the update.